### PR TITLE
Some changes before release.

### PR DESCRIPTION
**Pull request by kriii**
Changes:
- Update `setup.py`.
- Setting `tox` for the unit test. (Excluded python3.7 because `typing.Literal` is new in python3.8.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
```
Comment (platfowner) [inline README.md:2]: Could you add some descriptions (e.g. one-line definition, how to install, how to test)?
A one-line description example: "A python version of ain-js [link to https://www.npmjs.com/package/@ainblockchain/ain-js]."
cc: @liayoo

Comment (kriii) [reply README.md:2]: Added some descriptions. Please check!

File: README.md [hunk 2]
```diff
@@ -1,6 +1,28 @@
 # ain-py
 
+A python version of ain-js [link to https://www.npmjs.com/package/@ainblockchain/ain-js].
```
Comment (platfowner) [inline README.md:3]: could you follow the md syntax?
e.g. [ain-js](ttps://www.npmjs.com/package/@ainblockchain/ain-js)

**Review by platfowner — APPROVED** [review]
LGTM with some comments (please address before merging).
Thanks!

File: README.md (hunk 2 above)
Comment (kriii) [reply README.md]: Addressed! Thanks!

**Comment by kriii** [comment]
Merged and [released](https://pypi.org/project/ain-py/). Thanks for the reviews!

**Pull request merged**